MAUI/Maps/legend.md

```diff
@@ -660,4 +660,169 @@ public class Model
 
 ![Legend position](images/legend/legend-position.png)
 
+## Legend icon customization
+
+You can customize the legend icon using the `IconType` and `IconSize` properties. The default value of `IconType` is [`ShapeType.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.ShapeType.html#Syncfusion_Maui_Core_ShapeType_Rectangle) and 12 is the default value of `IconSize`.
```

Comment:
Customize the legend icon using the `IconType` and `IconSize` properties. The default value of the `IconType` is [`ShapeType.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.ShapeType.html#Syncfusion_Maui_Core_ShapeType_Rectangle) and 12 is the default value of the `IconSize.`